WORKSPACE

```diff
@@ -141,8 +141,17 @@ apple_support_dependencies()
 ##############################################################################
 # Java
 ##############################################################################
+# Java microgenerator.
+_java_microgenerator_version = "3f098c4e21daa7b211cd43e6c4af03d627e754e4"  # Will change often in dev.
+
+http_archive(
+    name = "com_google_api_generator",
```

Comment:
I see you are following the same naming convention used in the monolith generator (the name of the bazel import corresponds to the name of the root java package of the generator). The other microgenerators simply use `gapic_generator_<language>` naming scheme, which has proven itself being less confusing to the users of this repo. Please consider using the same naming as the other microgenerators.